src/main/java/org/cbioportal/service/impl/ClinicalDataServiceImpl.java

```diff
@@ -27,7 +32,9 @@ public class ClinicalDataServiceImpl implements ClinicalDataService {
     @Autowired
     private ClinicalAttributeService clinicalAttributeService;
     @Autowired
-    private ClinicalAttributeUtil clinicalAttributeUtil ;
+    private ClinicalAttributeUtil clinicalAttributeUtil;
+    @Autowired
+    private PaginationCalculator paginationCalculator;
```

Comment:
See above.
If the PaginationCalculator is not a component you won't have to bring it in this way.

Reply:
@JREastonMarks  I agree. However, I merely changed and renamed the exisiting OffsetCalculator class. I would propose to change this in a followup pull request.

Reply:
@JREastonMarks  I agree. However, I merely changed and renamed the exisiting OffsetCalculator class. I would propose to change this in a followup pull request.